### ISSUE TITLE
yarn config needs sudo on macos

### ISSUE THREAD

**Issue by JCMais** [CircleCI-Public/node-orb#8]
### Orb version

1.1.0

### What happened

Running `yarn config set cache-folder ~/.cache/yarn` now needs sudo. It's weird for it to ask for sudo since it should be installed for the current user.

This started happening after upgrading from 1.0.1 to 1.1.0 to get the fixes from https://github.com/CircleCI-Public/node-orb/issues/5

### Expected behavior

Should run without sudo(?)

If that is expected please let me know.

**Comment by JCMais** [comment]
So I just found that just a `install` needs sudo too, I don't think we should be running install with sudo:
```
yarn install --frozen-lockfile --cwd=.
^D^Dyarn install v1.16.0
error An unexpected error occurred: "EACCES: permission denied, scandir '/Users/distiller/.config/yarn/link'".
info If you think this is a bug, please open a bug report with the information provided in "/Users/distiller/project/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Exited with code 1
````

**Comment by iynere** [comment]
hmm, weird, it's certainly not _installed_ with sudo.... we'll investigate, ty @JCMais

**Comment by iynere** [comment]
hmmm, i'm not able to reproduce @JCMais :

https://circleci.com/gh/iynere/sandbox/640

can you send a link to the job where you saw this ?

also, you might try something like this to fix : 

https://github.com/yarnpkg/yarn/issues/1806#issuecomment-274078814

**Comment by JCMais** [comment]
Running `sudo chown -R $USER:$GROUP ~/.config` fixed the issue, kinda weird.

I'm closing this since I don't have time to dig into it or to check why it was happening.

**Comment by iynere** [comment]
@JCMais well, for now, I can easily add that line to the orb code, so you won't have to keep running it 🙂 thanks !

**Comment by iynere** [comment]
patched in `circleci/node@1.1.2`

https://github.com/CircleCI-Public/node-orb/releases/tag/v1.1.2
